src/main/resources/templates/bash/test/test_helper/common-setup.bash

```
@@ -0,0 +1,34 @@
+_common_setup() {
+    bats_load_library "bats-support"
+    bats_load_library "bats-assert"
+    bats_load_library "bats-file"
+
+    PROJECT_ROOT="$( cd "$BATS_TEST_DIRNAME/.." >/dev/null 2>&1 && pwd )"
+    ASSIGNMENT_ROOT="$PROJECT_ROOT/${studentParentWorkingDirectoryName}"
```

Shellcheck (0.10.0)

[warning] studentParentWorkingDirectoryName is referenced but not assigned.

(SC2154)